schema/tests/architecture-dependent.rs

```diff
@@ -0,0 +1,24 @@
+use impls::impls;
+use iroha_schema::IntoSchema;
+use parity_scale_codec::{Decode, Encode};
+
+#[test]
+fn usize_isize_not_into_schema() {
+    // The architecture-dependent
+    assert!(!impls!(usize: IntoSchema));
+    assert!(!impls!(isize: IntoSchema));
+
+    // use serde::Serialize;
+    //
+    // assert!(!impls!(usize: Serialize));
+    // `usize` should be `Serialize`.
+
+    // But not `Encode`/`Decode`.
+    assert!(!impls!(usize: Encode | Decode));
+    assert!(!impls!(isize: Encode | Decode));
```

Comment:
it's fine at the moment, but if parity changes their mind the test would start failing. Do you think it makes sense to verify external lib working?

Reply:
I think I'd like to keep this test